cypress/support/commands.js

```diff
@@ -100,23 +99,65 @@ Cypress.Commands.add('boardDeleteApi', function(key, token) {
             cy.wrap(boardId).as('boardId');
         });
     });
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+
+    Cypress.Commands.add('getWorkSpacedId', function (key,token, workSpaceName) {
+        cy.request({
+            url: `${Cypress.env('urlApi')}/members/me/organizations?key=${key}&token=${token}`,
+            method: "GET",
+        }).then((response) => {
+            expect(response.status).to.eq(200);
+            let workSpaceId = getWSId(response.body, workSpaceName);
+            console.log(response.body);
+            cy.wrap(workSpaceId).as('workSpaceId');
+            console.log(workSpaceId);
+        });
+    });
+    
+    Cypress.Commands.add('workSpaceDeleteApi', function(key, token) {
+        cy.request({
+        url: `${Cypress.env('urlApi')}/organizations/${this.workSpaceId}?key=${key}&token=${token}`,
+        method: 'DELETE',
+        }).then(response=>{
+                expect(response.status).to.eq(200);
+            });
+    });
+
+    Cypress.Commands.add('workSpaceCreateApi', function(key, token, workSpaceName) {
+        cy.request({
+            url: `${Cypress.env('urlApi')}/organizations/?displayName=${workSpaceName}&key=${key}&token=${token}`,
+            method: 'POST',
+            }).then(function(response){
+                expect(response.status).to.eq(200);
+                const workSpaceId = response.body.id;
+                cy.wrap(workSpaceId).as('workSpaceId');
+                });
+    });
+
+    // FUNCTIONS 
+function getId(arr, boardName) {
```

Comment:
This util functions should be in another file and just be imported here, this file should only contain cypress Commands.